rubin_sim/utils/cameraFootprint.py

```diff
@@ -21,50 +20,54 @@ class LsstCameraFootprint(object):
     """
     Class to provide the capability for identifying observations within an LSST camera footprint.
     """
-    def __init__(self, obsRA='fieldRA', obsDec='fieldDec'):
-        self.obsRA = obsRA
-        self.obsDec = obsDec
-        filename = os.path.join(get_data_dir(), 'maf/fov_map.npz')
-        _temp = np.load(filename)
+    def __init__(self, units='degrees', footprint_file=None):
+        if footprint_file is None:
+            footprint_file = os.path.join(get_data_dir(), 'maf', 'fov_map.npz')
+        _temp = np.load(footprint_file)
+        # Units refers to the incoming ra/dec values in the _call__ method
+        # Internally, radians are used to calculate the footprint
+        self.units = units
         self.camera_fov = _temp['image'].copy()
         self.x_camera = np.radians(_temp['x'].copy())
         _temp.close()
         self.plate_scale = self.x_camera[1] - self.x_camera[0]
         self.indx_max = len(self.x_camera)
 
-    def __call__(self, ephems, obsData, epoch=2000.0, rmax=2.1, timeCol='observationStartMJD',
-                 rotSkyCol='rotSkyPos'):
+    def __call__(self, obj_ra, obj_dec, boresight_ra, boresight_dec, boresight_rotSkyPos):
         """Determine which observations are within the actual camera footprint for a series of observations.
 
         Parameters
         ----------
-        ephems : np.recarray
-            Ephemerides for the objects, with RA and Dec as 'ra' and 'dec' columns (in degrees).
-        obsData : np.recarray
-            Observation pointings, with RA and Dec as 'ra' and 'dec' columns (in degrees).
-            The telescope rotation angle should be in 'rotSkyPos' (in degrees), and the time of each
-            pointing should be in the 'expMJD' column.
-        rmax : float (2.1)
-            The maximum radius from the center of the field to consider
-        epoch: float, optional
-            The epoch of the ephemerides and pointing data. Default 2000.0.
+        obj_ra : `np.ndarray`
```

Comment:
I don't know if the fancy linking done in creating of the documentation will be confused by using the `np.recarray` abbreviation rather than the full `numpy.recarray`. This should be checked.

Reply:
I think that I figured out maybe this problem I saw is due to not importing things in the conf.put of the doc build stage. I’ll check.

Reply:
It still doesn't work right, but neither does numpy.ndarray .. numpy.recarray seems like it's also failing, but I'm less sure. 
Going with this for now, but should check in with dm-docs.